widgets/versionlist/partials/_widget-contents.htm

```diff
@@ -2,6 +2,7 @@
     <div class="sidepanel-content-header">
         <?php if ($pluginVector): ?>
             <span data-localization-key="<?= e($pluginVector->getPluginName()) ?>" data-plugin="<?= e($pluginVector->pluginCodeObj->toCode()) ?>"><?= e(trans($pluginVector->getPluginName())) ?></span>
+            <?=$this->makePartial('sort'); ?>
```

Comment:
@filipac Space between `=` and `$`.
```suggestion
            <?= $this->makePartial('sort'); ?>
```

Reply:
@filipac this one still needs to be addressed

Reply:
done.